js/src/components/lineage/LineageView.tsx

```diff
@@ -598,6 +599,14 @@ export function PrivateLineageView(
       // Skip the following logic if the run result is not open
       return;
     }
+    if (
+      !runResultType ||
+      ["query_diff", "query", "row_count"].find((x) => x === runResultType)
```

Comment:
Hi Kent,

Would it be better to use `[xxx].includes`?
We could get a boolean directly.

Reply:
Thx suggestion!